internal/app/machined/internal/install/install.go

```diff
@@ -113,7 +114,14 @@ func RunInstallerContainer(disk, platform, ref string, configBytes []byte, reg c
 		oci.WithHostHostsFile,
 		oci.WithHostResolvconf,
 		oci.WithParentCgroupDevices,
-		oci.WithPrivileged,
+		oci.WithCapabilities(capability.AllGrantableCapabilities()),
```

Comment:
I had to expand `Privileged` into explicit list, as in our case listing "all" capabilities for the reason that machined has different permitted and bounding capabilities.